### PR TITLE
added unit test for pageNavigatorCustom class

### PR DESCRIPTION
**Pull request by mysticeti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,4 +40,18 @@ Modify `web/index.html` to specify your API key for Web.
...
 To provide compile-time variables use `--dart-define` during `flutter run` or `flutter build`. To pass multiple key-value pairs just use `--dart-define` multiple times.
-`flutter run --dart-define=ACCESS_TOKEN_MAPBOX --dart-define=VARIABLE=VARIABLE_VALUE`
\ No newline at end of file
+`flutter run --dart-define=ACCESS_TOKEN_MAPBOX --dart-define=VARIABLE=VARIABLE_VALUE`
+
+## Quality Assurance
+#### Run Unit Test
+##### Android Studio
+- Right-click on pageNavigatorCustom_test.dart file
+- Press Run button to run the test.
```
Comment (joykuotw) [inline README.md:49]: Use numbered lists

File: README.md [hunk 2]
```diff
@@ -40,4 +40,18 @@ Modify `web/index.html` to specify your API key for Web.
...
+
+##### Visual Studio Code
+- Open the pageNavigatorCustom_test.dart file
+- Select the Run menu
+- Click the Start Debugging option
```
Comment (joykuotw) [inline README.md:54]: Use numbered lists

File: README.md [hunk 3]
```diff
@@ -40,4 +40,18 @@ Modify `web/index.html` to specify your API key for Web.
...
+
+##### Terminal & editor
+`flutter test test/pageNavigatorCustom_test.dart`
```
Comment (joykuotw) [inline README.md:57]: Use `flutter test` is enough

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joykuotw) [inline README.md:49]: Follow this I am getting error in Android Studio. Is it working on yours?

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mysticeti) [reply README.md:49]: updated in the next commit

File: README.md (hunk 2 above)
Comment (mysticeti) [reply README.md:54]: updated in the next commit

File: README.md (hunk 3 above)
Comment (mysticeti) [reply README.md:57]: updated in the next commit

File: README.md (hunk 1 above)
Comment (mysticeti) [reply README.md:49]: I am not seeing any errors. Make sure to run the pageNavigatorCustom_test.dart file. Not the main.dart file

Comment (joykuotw) [reply README.md]: This is what I get. Screenshot 2021-05-27 at 12 46 16

Comment (joykuotw) [reply README.md]: Resolved. I run the pageNavigatorCustom_test file itself instead of running the test on it.

File: README.md [hunk 4]
```diff
@@ -40,4 +40,18 @@ Modify `web/index.html` to specify your API key for Web.
...
 To provide compile-time variables use `--dart-define` during `flutter run` or `flutter build`. To pass multiple key-value pairs just use `--dart-define` multiple times.
-`flutter run --dart-define=ACCESS_TOKEN_MAPBOX --dart-define=VARIABLE=VARIABLE_VALUE`
\ No newline at end of file
+`flutter run --dart-define=ACCESS_TOKEN_MAPBOX --dart-define=VARIABLE=VARIABLE_VALUE`
+
+## Quality Assurance
+#### Run Unit Test
+##### Android Studio
+1. Right-click on pageNavigatorCustom_test.dart file
+2. Press Run button to run the test.
```
Comment (joykuotw) [inline README.md:49]: Press the Run `tests in pageNavigatorCustom_test.dart` option.

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (mysticeti) [reply README.md]: Updated the instruction in the next commit

**Review by joykuotw — APPROVED** [review]

**Pull request merged**